www/include/configuration/configObject/contact/formContact.php

```diff
@@ -92,10 +92,10 @@
     $cct["contact_svNotifCmds"] = array();
     $cct["contact_cgNotif"] = array();
 
-    $DBRESULT = $pearDB->query("SELECT * FROM contact WHERE contact_id = '" . intval($contact_id) . "' LIMIT 1");
-    $cct = array_map("myDecode", $DBRESULT->fetchRow());
+    $dbResult = $pearDB->query("SELECT * FROM contact WHERE contact_id = '" . intval($contact_id) . "' LIMIT 1");
```

Comment:
Please use **bindValue** to set **contact_id** value
```suggestion
    $dbResult = $pearDB->prepare("SELECT * FROM contact WHERE contact_id = :contact_id LIMIT 1");
    $dbResult->bindValue(':contact_id', $contact_id, \PDO::PARAM_INT);
    $dbResult->execute();
```